gnat2goto/driver/tree_walk.adb

```diff
@@ -4286,12 +4289,9 @@ package body Tree_Walk is
    -------------------------
 
    procedure Do_Withed_Unit_Spec (N : Node_Id) is
-      Unit_Name : constant String := Get_Name_String (Get_Unit_Name (N));
    begin
-      if Defining_Entity (N) = Stand.Standard_Standard or else
-        Unit_Name = "system%s"
-      then
-         --  At the moment Standard or System are not processed: TODO
+      if Defining_Entity (N) = Stand.Standard_Standard then
+         --  At the moment Standard is not processed: TODO
```

Comment:
Maybe report as unsupported?

Reply:
+1 Yes, please report things that are "TODO" as unsupported features - this might not be important for UKNI, but will be important when gnat2goto is applied to new codebases that we might not have access to. In those cases we really would like to get the most complete list of unsupported features we can.

Reply:
This line is also why the regression tests are failing.

Reply:
This mod has now been removed from this branch.  It now only deals with qualified expression.